examples/tokenizer_training.ipynb

```diff
@@ -375,6 +375,24 @@
     "# !huggingface-cli login"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are almost there, it is also necessary that you have `git lfs` installed and that you have configured your git. You can do it directly from this notebook by uncommenting the following cells:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !sudo apt-get install git-lfs\n",
+    "# !git config --global user.email \"you@example.com\"\n",
+    "# !git config --global user.name \"Your Name\""
+   ]
+  },
```

Comment:
I propose to add this command `!sudo apt-get install git-lfs` because before without it I got the error:
```
OSError: Looks like you do not have git-lfs installed, please install. You can install from https://git-lfs.github.com/. Then run `git lfs install` (you only have to do this once).
```
I propose to add this command `!git config --global user.email "you@example.com" & git config --global user.name "Your Name"` because before I got this error:
```
OSError: 
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'root@dc8dbe513cdf.(none)')
```